.markdownlint-cli2.cjs

```diff
@@ -0,0 +1,19 @@
+const githubMarkdownOpinions = require('@github/markdownlint-github')
+
+const options = githubMarkdownOpinions.init({
+  // Disable rules we don't currently care to enforce pertaining to stylistic things.
+  'line-length': false,
+  'blanks-around-headings': false,
+  'blanks-around-lists': false,
+  'no-trailing-spaces': false,
+  'no-multiple-blanks': false,
+  'no-trailing-punctuation': false,
+  'single-trailing-newline': false,
+  'ul-indent': false,
+  'no-hard-tabs': false
```

Comment:
From PR description: 

Some of markdownlint's defaults are related to stylistic things like `line-length` and `blanks-around-headings`. I don't know how much value Primer gets in enforcing these things so I decided to turn them off. There's also a lot to resolve.

I made sure the rules pertaining to accessibility are kept on since these provide a lot of value in encouraging accessibility practices.